src/main/kotlin/lotto/domin/Lotto.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domin
+
+import lotto.util.LottoNumberGenerator
+
+class Lotto(
+    private val payment: Int,
+    private val lottoNumberGenerator: LottoNumberGenerator
+) {
+
+    fun issuanceCount(): Int {
+        return payment.div(LOTTO_PRICE)
+    }
+
+    fun issuance(): List<Int> {
+        return this.lottoNumberGenerator.numberSet()
+    }
+
+    companion object {
+        const val LOTTO_PRICE = 1000
```

Comment:
```suggestion
        const val LOTTO_PRICE = 1_000
```
언더바를 활용하여 천단위로 끊어주면, 좀 더 잘 읽혀요. 😄